sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3229,6 +3229,15 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val CHUNK_BASE64_STRING_ENABLED = buildConf("spark.sql.legacy.chunkBase64String.enabled")
+    .internal()
+    .doc("Whether to truncate string generated by the `Base64` function. When true, base64" +
+      " strings generated by the base64 function are chunked into lines of at most 76" +
+      " characters. When false, the base64 strings are not chunked.")
+    .version("3.5.2")
```

Comment:
IIUC, if we start to backport SPARK-47307, it will go to Apache Spark 3.4.4 together, right? In that case, I'm curious if 3.5.2 is correct.

Reply:
Hmm. Got it. I saw this comment from Apache Spark 3.5.2 release manager, @yaooqinn .
- https://github.com/apache/spark/pull/47303/files#r1675645743

If then, are we going to update these values from `master` and `branch-3.5` when we do the release of Apache Spark 3.4.4? I'm fine if we are going to do in that way.

Reply:
Thank you @dongjoon-hyun. 

Not related to this PR, maybe we shall add multiple fixed version in this field, such as `3.4.4, 3.5.2, 4.0.0`

Reply:
Theoretically, it's possible, but it will enforce us to update all the existing configurations and documentations. So, we had better not to because it could be too much.